sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -332,6 +332,43 @@ public void disablePersisted() {
         verify(channel, never()).addListener(any(Channel.Listener.class));
     }
 
+    @Test
+    public void notSendingLogsOnPause() {
+
+        /* Before start it does not work to change state, it's disabled. */
```

Comment:
then we should have a test for what happens if calling before start and happens when calling pause/resume while disabled.

Reply:
I'll add the one for start but we already have tests for being disabled [here](https://github.com/Microsoft/AppCenter-SDK-Android/pull/803/files#diff-fb7e581dbd71041b785eca0c6a15b437R807)

Reply:
No thats not testing that when Analytics disabled it's not even trying to call channel APIs.